src/main/steps/applicant/yourhearings/hearings/content.ts

```diff
@@ -1,10 +1,30 @@
+import { applyParms } from '../../../../steps/common/url-parser';
 import { TranslationFn } from '../../../../app/controller/GetController';
 import { generateContent as yourhearingshearingscontent } from '../../../common/yourhearings/hearings/content';
+import { APPLICANT_TASK_LIST_URL, FETCH_CASE_DETAILS } from '../../../../steps/urls';
+import { CaseType, PartyType } from '../../../../app/case/definition';
 
 export { form } from '../../../common/yourhearings/hearings/content';
 export const generateContent: TranslationFn = content => {
   const hearingsContent = yourhearingshearingscontent(content);
+  const request = content.additionalData?.req;
+  const caseData = request.session.userCase;
+  if (content.additionalData?.req.session.userCase.caseTypeOfApplication === 'C100') {
```

Comment:
Content.ts should have the logic to build the case data according to design.

Reply:
Yeah this for link on where support during case should be navigated to . And it is given on content.ts itself.